### ISSUE TITLE
Opening Steam Client Causes Kernel Panic

### ISSUE THREAD

**Issue by abranchiate** [ValveSoftware/steam-for-linux#7244]
#### Your system information

* Steam client version (build number or date): steam 1.0.0.63-1
* Distribution (e.g. Ubuntu): Arch Linux (5.7.7-arch1-1)
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Opening the Steam Client and waiting a few minutes causes a Core Dump and a Kernel Panic. The Core Dump looks as follows:

Jul 05 12:49:02 archlinux systemd-coredump[15660]: Process 15647 (vulkandriverque) of user 1000 dumped core.
Process 15647 (vulkandriverque) of user 1000 dumped core.                                                   
Stack trace of thread 15647:
#0  0x00007fb7e7434355 raise (libc.so.6 + 0x3c355)
#1  0x00007fb7e741d853 abort (libc.so.6 + 0x25853)
#2  0x00007fb7e741d727 __assert_fail_base.cold (libc.so.6 + 0x25727)
#3  0x00007fb7e742c936 __assert_fail (libc.so.6 + 0x34936)
#4  0x00005630bd9ed467 n/a (/home/xxx/.local/share/Steam/ubuntu12_64/vulkandriverquery + 0xd6467)
#5  0x00005630bd9d4277 n/a (/home/xxx/.local/share/Steam/ubuntu12_64/vulkandriverquery + 0xbd277)
#6  0x00007fb7e741f002 __libc_start_main (libc.so.6 + 0x27002)
#7  0x00005630bd9d40ea n/a (/home/xxx/.local/share/Steam/ubuntu12_64/vulkandriverquery + 0xbd0ea)

#### Steps for reproducing this issue:

1. Open Steam Client (Runtime) and wait a few minutes

#### Other
Everything worked perfectly fine until i performed a full system upgrade approximately a week ago. It first occurred, after i started some downloads. Downloading is still incomplete. 

I would guess, the problem is related to solved <a href="https://github.com/ValveSoftware/steam-for-linux/issues/6230">Issue #6230</a> or <a href="https://github.com/ValveSoftware/steam-for-linux/issues/7160">Issue #7160</a> and has something to do with my installed drivers. 
There are some Issues regarding Arch Linux and Core Dumps since mid May 2020. This might be connected.

I have got the following packages installed:

<ul>
<li>steam 1.0.0.63-1</li>
<li>lib32-nvidia-390xx-utils 390.132-1</li>
<li>lib32-vulkan-icd-loader 1.2.145-1</li>
<li>nvidia-390xx-utils 390.132-2</li>
<li>vulkan-icd-loader 1.2.145-1</li>
</ul>

**Comment by kisak-valve** [comment]
Hello @abranchiate, no OpenGL application, including the Steam client should be capable of crashing the entires system, which hints that you've encountered a video driver or kernel issue. What video card are you using?

Since you're using the nvidia 390 series, I suspect you're using a Fermi generation video card which does not support vulkan and are expected to fail the basic vulkan initialization test, and vulkandriverquery failing is a red herring.

**Comment by abranchiate** [comment]
Hello,

thank you for your fast reply. I am using a Lenovo G780 Notebook, which has a **Nvidia GeForce GT 630M**. Yes, it is a **Fermi architecture** video card.
Can i support you with any more information like logs etc. to shed light to the issue?

**Comment by Newbytee** [comment]
> Can i support you with any more information like logs etc. to shed light on the issue? Or is this a problem that i need to address to the Kernel/Driver maintainers?

You could try seeing if the issue persists with Nouveau. If it does, this is probably NVIDIA's bug, if not, Linux' bug.

**Comment by abranchiate** [comment]
Hello,
thank you very much for your support. I will close this Issue, because it can not to be caused by the steam client.

FYI:
I tried to capture the Kernel Panic Message for further research, which worked by redirecting systemd-output to tty, but it was too long to be read completely. I tried to generate the Issue again with enabled Kernel Parameter _pause_on_oops=10_, but could not recreate the problem.
I noticed, that my remaining Downloads just completed after the last Kernel Panic. Since then, everything is running fine, so i suspect the Issue has something to do with Downloads, because it first occurred, after i started some, too. 
It didn't happen, while the client was minimized, so, for now, i will stick to minimizing the client, when downloading.
